### PR TITLE
Simple mind mutation

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Content "A mutation that reduces INT and removes psychology-related mutations"

#### Purpose of change
Bugs, snails, lobsters... simple creatures that remain functional when the world changes.

Plants. They feel. They grow. They don't think.

#### Describe the solution
Simple mind mutation: INT -1, removes mood-related mutations and kaluptic psychosis. It applies to insect, gastropod, crustacean, and plant.

#### Describe alternatives you've considered
Frogs might be also included. They're left out because they are vertebrate.

#### Testing
Game loads. Activate mood swings. Activate simple mind. INT lowered, mood swings gone.

### CODE REVIEW

**Comment by fairyarmadillo** [comment]
Giving this ATTITUDE means that Plant will eternally flipflop between gaining and losing seasonal affective disorder, which they are meant to have due to their systems becoming depressed by seasonal dormancy.

The game also seems to avoid handing out big stat debuffs as traits like this, and for good reason. Characters often start with single-digit intelligence. If someone picked 6 at chargen, this puts them at a base of 3, and they're going to spend a lot of their time debuffed to 1 or 0 by pain or other common effects. At that point the game would be sort of unplayable.

And a -3 intelligence penalty with no upsides should probably not be a 0 point mutation.

**Comment by fairyarmadillo** [comment]
If I could make a suggestion, maybe changing this to an enchantment that multiplies the character's intelligence by 0.8 and gives them immunity or resistance to having their intelligence debuffed by pain might be a better approach.

**Comment by Venera3** [comment]
Not sure on either insect nor spider getting hit with it - both have very intelligent species, and you already make an exception for ceph despite nautiluses, well, nautilusing. Frog or Beast would feel a better fit than those two, but that might be my invert favoritism at play.

Intelligence (in the sense of operative learning) has nothing to do with pain, and we already have pain resistance as its own trait line even if inverts didn't feel pain (they do).

**Comment by bateau-de-coal** [comment]
1.Why is it both INT and ATTITUDE?: Primitive or none neural system. Thus low INT and the lack of advanced psychology.
2. Is -3 too much?: -2 could be more reasonable than -3.
3. Seasonal affective disorder: a solution could be making seasonal affective disorder not an ATTITUDE, as it is not a psychological problem. The other ATTITUDES look fine with this.
4. Are invertebrates smart?: except for cephalopod, most invertebrates can't solve problem via reasoning. They don't rely on their brain.
5. Pain: I don't plan to include pain in this.

**Comment by fairyarmadillo** [comment]
> Intelligence (in the sense of operative learning) has nothing to do with pain, and we already have pain resistance as its own trait line even if inverts didn't feel pain (they do).

I didn't mean to imply they don't, but you do see arthropods behaving somewhat normally despite suffering catastrophic injury fairly often, like mating while being eaten or (seemingly) calmly surrendering a leg to a predator and walking away. Mammals seem to be less capable of that sort of thing.

Beast already gets a -3 intelligence debuff from Apex Predator. Maybe ectotherms could get a cold-based int debuff? That'd incentivize people to set up little habitats for themselves in the winter so they could keep reading and crafting.

**Comment by bateau-de-coal** [comment]
Thanks for the tip about making SAD purifiable.

It would be difficult to only partially cancel Chemical Imbalance. From my experience, CI has very minor effects. It is mostly a flavor trait.

Simple Mind specifically wouldn't affect cephalopod.

JSON data shows that Numb only adds a flag. What does it do actually?

**Comment by bateau-de-coal** [comment]
Numb is an interesting trait. I prefer not include it in this though. Food is a big factor that interacts with morale. Morale boost from good food is very real for invertebrates.

**Comment by Consoleable** [comment]
Spiders should not be on this list. There are a lot of dumb spiders, but there are a lot of dumb fish too. Spiders can be pretty good tool users, and many species have excellent, if monomaniacal intelligence(like jumping spiders, who have some of the best planning skills in the animal kingdom) and the tree already suffers debuffs. Animal intelligence can be very subjective(due to biases in how humans judge animal reactions and visual responses) and it's a pretty broad stat in game. 

The mutations are definitely a gamified part of cdda. What a zooiform human mutant should end up like is pretty subjective. If mutation categories lose intelligence because that makes them somehow resemble that species, shouldn't more mutation trees have decreased size? Most of the mutation trees reference animals which are much smaller than humans, yet have no size change or even enhanced size(crabs, lizards, bats, frogs etc.) why is a crustacean a large mutant? why do lizards get good hearing? Shouldn't all mutation trees, barring elf, medical, and alpha get this trait?

Since there is obvious gamification of the mutation lines, I think there should be gamification justifications of changes like this, especially if this is un-purifiable or part of the threshhold tree of mutation lines. Is this mutation really (0) and not negative? Are the lines it's a part of very good otherwise? Or is this a "smooth brain" meme in the form of a PR? Cancelling other possible negative mutations isn't really enough imo. Maybe a built in mood buff, or focus loss prevention?

**Comment by bateau-de-coal** [comment]
How about -1 point, INT -1, removing Spider from "category"? Compare "Smart": +1 point, INT +1.

**Comment by bateau-de-coal** [comment]
Does INT-1 not fit for insect? Some examples of smart insects may be needed here.

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -466,7 +467,6 @@
     "points": -1,
     "description": "You tend to have bouts of depression during the fall and especially winter.  Taking antidepressants can help.",
     "starting_trait": true,
-    "types": [ "ATTITUDE" ],
```
Comment (Fris0uman) [inline data/json/mutations/mutations.json:469]: Why editing the type of another unrelated trait here?

Comment (bateau-de-coal) [reply data/json/mutations/mutations.json:469]: fairyarmadillo commented May 8, 2024 Giving this ATTITUDE means that Plant will eternally flipflop between gaining and losing seasonal affective disorder 
bateau-de-coal commented May 8, 2024 Seasonal affective disorder: a solution could be making seasonal affective disorder not an ATTITUDE, as it is not a psychological problem.
Kamejeir commented May 8, 2024 Seasonal Affective Disorder can have its category removed, but don't forget to add it to the list of stuff Human Mind cancels or tests will give you a hard time.

**Comment by bateau-de-coal** [comment]
Splitting it into 2 mutations: Stupid: INT-1, and Indifference: removes ATTITUDE and kaluptic psychosis, clamps Morale. Good?

**Comment by Fris0uman** [comment]
I don't think it should affect kaluptic psychosis

**Review by kevingranade — CHANGES_REQUESTED** [review]
I'm not following the rationale for this at all, it links a token int debuff with "you're now immune to nervous disorders", except the list of disorders is very limited.

If you're committing to "animal level intelligence free from nervous disorders" I feel like we'd be talking about losing access to crafting and speech,  not a tiny loss of learning speed.

**Comment by bateau-de-coal** [comment]
So nothing affecting ATTITUDE and kaluptic psychosis. Then only Stupid: INT-1.

**Comment by fairyarmadillo** [comment]
I don't think they should even get that.

**Comment by bateau-de-coal** [comment]
It is obvious that some of the invertebrates have simpler brains. What is the reason to not show this in their mutation tree?

**Comment by Kamejeir** [comment]
> It is obvious that some of the invertebrates have simpler brains. What is the reason to not show this in their mutation tree?

Kevin told you the direction to take. Human hybrids being in possession of "simpler brains" would amount to more than a "Stupid" mutation.

**Comment by bateau-de-coal** [comment]
INT-1 looks reasonable enough, until some examples are provided, of how insects, snails, crustaceans, and plants, use their intelligence to solve problem.

**Comment by bateau-de-coal** [comment]
@fairyarmadillo What is confusing about this? Just to clarify: we're now only talking about a trait Stupid: Int-1, nothing else. It won't affect ATTITUDE, won't remove any psychosis.

It makes sense that these mutants are not free from neural disorders, but there is little reason that their intelligence is not even slightly affected.

**Comment by osuphobia** [comment]
I think you didn't understand what kevin was saying.

>If you're committing to "animal level intelligence free from nervous disorders" I feel like we'd be talking about losing access to crafting and speech, not a tiny loss of learning speed.

That is, -1 INT is only "a little bit silly", but to remove psychology-related mutations, you need to give them "a brain that is less functional", like the lack of ability to craft or speak, as a trade-off.

**Comment by kevingranade** [comment]
Sure if you want a token int penalty (or a more significant int penalty) go ahead.

**Pull request closed without merging**